support-frontend/assets/helpers/abTests/abtestDefinitions.ts

```diff
@@ -15,6 +15,8 @@ export const pageUrlRegexes = {
 			'^(?!(?:/subscribe/(paper|weekly)/checkout$))(?:/(uk|us|ca|eu|nz|int))?/(checkout|one-time-checkout|contribute|thankyou|thank-you)(/.*)?$',
 		usLandingPageOnly: '/us/contribute$',
 		genericCheckoutOnly: '(uk|us|au|ca|eu|nz|int)/checkout|thank-you(/.*)?$',
```

Comment:
I don't think there's any impact here, but maybe we should tighten the `genericCheckoutOnly`/`oneTimeCheckoutOnly` regex as the `thank-you` route is the same. EG. I'm pretty sure a user going through the generic checkout and landing on the TY page would then get bucketed in any test targeting `oneTimeCheckoutOnly` on the TY page view.

Reply:
We'd probably have to add something to the `oneTimeCheckoutOnly` regex to look for the absence of a `product=` query string param, and the opposite for the `genericCheckoutOnly` regex, looking for the presence of the `product=` query string param.

Reply:
@paul-daniel-dempsey I also think we can do this in a follow-up PR

Reply:
Have updated the last commit to attempt this regx change (looks for `?contribution` or `?product` after `thank-you` in url). would be worth going through on Monday whether its good enough  if you have time. Will not put PR in until we have checked this out this approach.